src/chainstate/burn/db/sortdb.rs

```diff
@@ -696,6 +722,21 @@ const SORTITION_DB_SCHEMA_4: &'static [&'static str] = &[
     );"#,
 ];
 
+// TODO(sbtc): Add tables for remaining sBTC ops
```

Comment:
Style nit: TODOs like this belong as github issues, not source code comments.

Reply:
Agreed. I will not merge until these TODO:s are either resolved or turned into issues.